.circleci/src/jobs/build_ios.yml

```diff
@@ -63,17 +63,17 @@ steps:
             command: bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE} build_config_name:integreat
             working_directory: native/ios
         - run:
-            name: Rename sourcemap
-            command: mv index.ios.bundle.map output/integreat.ios.bundle.map
-            working_directory: native/ios
+            name: Create sourcemap
```

Comment:
I'm not sure whether we want to additinally run `react-native bundle` this takes about 1-2 minutes and slows down the CI. We can also just pass a parameter to the xcode script which then creates a source map.

Reply:
Could you help me with that? How would one do that?

Reply:
In fastlane there is a environment variable EXTRA_PACKAGER_PARAMS or similar. You can just append there " --sourcemap-output ios/output/integreat.ios.bundle.map"

This works for ios and android

Reply:
Ahh found the issue. Afaics, the sourcemap is generated automatically for android. For ios just the path was wrong...